### PR TITLE
Fixed coverage reports

### PR DESCRIPTION
**Pull request by jhellar**
# Motivation
https://issues.jboss.org/browse/RHMAP-14680

# Changes
- replaced turbo with mocha for code coverage
- added grunt to dev-dependencies (needed with npm version >=3)

# Steps to verify
- `npm install`
- then these commands should work:
  - `grunt serve`
  - `grunt test`
  - `grunt coverage`

### CODE REVIEW

**Comment by jhellar** [comment]
retest this please

File: Gruntfile.js [hunk 1]
```diff
@@ -83,22 +75,7 @@ module.exports = function(grunt) {
...
-          stdout: true,
-          stderr: true,
-          failOnError: true
-        },
-        command: [
-          'rm -rf coverage cov-accept',
-          'env NODE_PATH=. ./node_modules/.bin/istanbul cover --dir cov-accept ./node_modules/.bin/turbo -- --setUp=test/accept/server.js --tearDown=test/accept/server.js test/accept',
-          './node_modules/.bin/istanbul report',
-          'echo "See html coverage at: `pwd`/coverage/lcov-report/index.html"'
+          'env NODE_PATH=. ./node_modules/.bin/istanbul cover --dir cov-unit ./node_modules/.bin/_mocha -- -u exports -R spec ./test/unit/test-static-route.js'
```
Comment (corinnekrych) [inline Gruntfile.js]: @jhellar what about test-main.js?
maybe we can test ./test/unit/test-*.js

Comment (jhellar) [reply Gruntfile.js]: yes, will fix that

**Comment by jhellar** [comment]
should be fixed now

**Comment by corinnekrych** [comment]
@jhellar main comment is lack of README instructions. I've PR your PR with https://github.com/jhellar/appforms-datasources-mongodb/pull/1

otherwise +1

**Comment by jhellar** [comment]
@corinnekrych I've merged your PR. Is this ready to be merged?

**Comment by jhellar** [comment]
Changes already merged.

**Pull request closed without merging**